modules/dataframe/src/properties/filterlistproperty.cpp

```diff
@@ -60,25 +60,28 @@ FilterListProperty::FilterListProperty(std::string_view identifier, std::string_
         p->setVisible(enable);
         return p;
     };
+    auto createBoolComposite = [](std::string_view identifier, std::string_view displayName) {
+        auto p = std::make_unique<BoolCompositeProperty>(identifier, displayName, true);
+        p->getBoolProperty()->setSerializationMode(PropertySerializationMode::All);
```

Comment:
This should be p->setSerializationMode(PropertySerializationMode::All);

Reply:
Why? Then the bool property (owned by the composite) would not be serialized.

Reply:
yes it will. you only need to set it on the property that is created dynamically. The bool child property is not dynamic ans is added in the constructor as usual...

Reply:
Might even be good if the ListProperty made sure all the Prefabs, has PropertySerializationMode::All

Reply:
I suppose this part is a little bit complicated ;)

Reply:
> yes it will. you only need to set it on the property that is created dynamically. The bool child property is not dynamic and is added in the constructor as usual...

But the constructor of the BoolComposite initializes the bool property by default with `false`, the `FilterListProperty` however sets the bool property to `true`.

Reply:
That is not relevant for the serialization mode.